bulletin/index.html

```diff
@@ -203,6 +203,26 @@ <h2 class="featured-article__title"> Engineering Robust Learning Environments Th
 	  </div>
 	</section>
 
+	<section>
+		<div class="featured-article__wrapper comments-acknowledge__wrapper">
```

Comment:
Why is this using the `featured-article__wrapper` class?

Reply:
Can we shorten the name of this section? I find `comments-acknowledge` rather long. What about `acknowledgements`?

Reply:
Agree. I decided to use "mentions" instead because this word encapsulate both comments and acknowledgements.

Reply:
I suggest merging the commits in this branch. Some issues have been addressed already.